web/l10n/en.json

```diff
@@ -141,6 +141,8 @@
     "positionDistance": "Distance",
     "positionRpm": "RPM",
     "positionFuel": "Fuel",
+    "positionPower": "Power",
```

Comment:
I was not sure about these two, but may they can be needed later. I would like to add a charts for battery and power (like speed, distance, etc.) Tried to analyse your code to find out if we need them, ReportChartTypes refers strings like this. Can you point me to the right direction to add more charts?